tests/chainer_tests/test_function_node.py

```diff
@@ -362,11 +362,9 @@ def check_mix_xp(self, xp):
         xp_x1 = xp.random.randn(2, 3).astype(numpy.float32)
         xp_x2 = xp.random.randn(2, 3).astype(numpy.float32)
         x2 = backend.to_chx(xp_x2)
-        y, = self.SimpleFunctionNode(xp).apply((xp_x1, x2))
 
-        assert isinstance(y.array, chainerx.ndarray)
-        chainerx.testing.assert_array_equal(
-            backend.CpuDevice().send(xp_x1 * xp_x2), y.array)
+        with self.assertRaises(TypeError):
+            y, = self.SimpleFunctionNode(xp).apply((xp_x1, x2))
```

Comment:
* Move instantiation of the function outside the scope.
* No need to assign the output to a variable.